### PR TITLE
Adopters and Layout Update + Fixed Minor Bugs

### PR DESCRIPTION
**Pull request by S-ayanide**
# Adopters and Layout Update + Fixed Minor Bugs

Signed-off-by: Sayan Mondal <sayan.mondal@mayadata.io>

### CODE REVIEW

File: src/components/footer/Footer.tsx [hunk 1]
```diff
@@ -58,30 +59,65 @@ const Footer: React.FC = () => {
                 <Links data={resources} showImage={true} internalLink={false} />
               </Row>
               <Row>
-                <Links
+                {/* <Links
```
Comment (arkajyotiMukherjee) [inline src/components/footer/Footer.tsx:62]: can we remove the comments

File: src/components/footer/data.ts [hunk 2]
```diff
@@ -41,35 +41,35 @@ const resources = {
...
-    {
-      name: "Zebrium",
-      url:
-        "https://github.com/litmuschaos/litmus/blob/master/adopters/zebrium.md",
-    },
-  ],
-};
+// const top_adopters = {
+//   title: "Top Adopters",
+//   links: [
```
Comment (arkajyotiMukherjee) [inline src/components/footer/data.ts:46]: are these not needed anymore?

File: src/components/pre-footer/PreFooterBottom.tsx [hunk 3]
```diff
@@ -1,103 +1,94 @@
...
 `;
 
-const CNCFLogoDiv = styled.div`
-  background: white;
-  box-shadow: 0px 8.55652px 24.9565px rgba(0, 0, 0, 0.06);
-  border-radius: 0.625rem;
-  padding: 1.375rem 1.875rem;
-  margin-bottom: 2rem;
+// const MayadataImage = styled.img`
+//   margin: ${(props) => (props.theme.screens.md ? "0 auto" : 0)};
```
Comment (arkajyotiMukherjee) [inline src/components/pre-footer/PreFooterBottom.tsx:13]: same here

File: src/components/pre-footer/PreFooterBottom.tsx [hunk 4]
```diff
@@ -1,103 +1,94 @@
...
+// `;
 
-  img {
-    margin: 0 auto;
-    width: ${props => (props.theme.screens.sm ? "90%" : "60%")};
-  }
-`;
+// const HorizontalWrap = styled.div`
+//   min-width: 25%;
+//   max-width: 100%;
```
Comment (arkajyotiMukherjee) [inline src/components/pre-footer/PreFooterBottom.tsx:19]: same

File: src/components/pre-footer/PreFooterBottom.tsx [hunk 5]
```diff
@@ -1,103 +1,94 @@
...
+      </BoundedContainer>
+      {/* <ResponsiveRow breakpoint="md" justifyContent="space-evenly">
+        <HorizontalWrap>
+          <SubHeading style={{ margin: "0.5rem 0" }}>
+            Originally created by
+          </SubHeading>
+          <a
+            rel="noopener noreferrer"
+            target="_blank"
+            href="https://mayadata.io/"
```
Comment (arkajyotiMukherjee) [inline src/components/pre-footer/PreFooterBottom.tsx:56]: same

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]
there's a lot of commented code....should we remove it altogether or will they be used again @S-ayanide

File: src/components/footer/data.ts (hunk 2 above)
Comment (S-ayanide) [reply src/components/footer/data.ts:46]: No, these commented lines will be removed. The design requirements have been updated.

**Review by amityt — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM

**Pull request merged**